client/src/store/index.js

```diff
@@ -2,10 +2,20 @@ import { createStore, combineReducers } from 'redux';
 
 import reducer from './reducer/reducer';
 import createSurveyReducer from './reducer/createSurveyReducer';
+import adminLoginReducer from './reducer/adminLoginReducer';
+import snackbarReducer from './reducer/snackbarReducer';
 import dashboardReducer from './reducer/dashboardReducer';
 
 const store = createStore(
-  combineReducers({ reducer, createSurveyReducer, dashboardReducer }),
+  combineReducers({
+    reducer,
+    createSurveyReducer,
+    adminLoginReducer,
+    snackbarReducer,
+    dashboardReducer,
+  }),
+  // eslint-disable-next-line no-underscore-dangle
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
```

Comment:
what is this for?

Reply:
Redux DevTools - vital for redux debugging.